stable/admin/templates/role.yaml

```diff
@@ -18,6 +18,7 @@ rules:
   resources:
   - deployments
   - statefulsets
+  - daemonsets
```

Comment:
do you still require this?

Reply:
We decided to display DaemonSets in `nuocmd --show-json get kubernetes-config` for diagnostic purposes although they are not used directly in KAA resync logic. I would propose to keep that in the role.